new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java

```diff
@@ -119,7 +122,7 @@
    */
   @BeforeAll
   public static void initAll(@Namespaces(2) List<String> namespaces) {
-
+    logger = ThreadSafeLogger.getLogger();
```

Comment:
you can static import ThreadSafeLogger.getLogger then just call getLogger() here like in other It* classes.